eslint.config.js

```
@@ -34,6 +34,16 @@ export default [
         { allowConstantExport: true },
       ],
       "react/prop-types": "off",
+      // Prevent console.log in production
+      "no-console": process.env.NODE_ENV === "production" ? "error" : "warn",
```

ESLint

[error] 'process' is not defined.

(no-undef)